x-pack/plugins/ml/public/application/components/data_grid/data_grid.tsx

```diff
@@ -310,6 +310,7 @@ export const DataGrid: FC<Props> = memo(
                       >
                         <EuiButtonEmpty
                           aria-checked={chartsVisible}
+                          role="checkbox"
```

Comment:
Instead of changing the role of this to a `checkbox` let's make this a toggle button instead 
```suggestion
                          aria-pressed={chartsVisible}
```

By not changing the role, we have a lot less risk in running into interop bugs between browsers, OSes, and assistive tech.

Reply:
Switched to `aria-pressed` in https://github.com/elastic/kibana/pull/88323/commits/55ed7968b20327dd9d5205bfb6c89d877aa3fd92